plc4j/tools/plc4x-server/pom.xml

```diff
@@ -88,15 +88,16 @@
 
     <dependency>
       <groupId>org.apache.plc4x</groupId>
-      <artifactId>plc4j-driver-simulated</artifactId>
+      <artifactId>plc4j-driver-plc4x</artifactId>
```

Comment:
Woudln't the plc4x driver be part of the all package?

Reply:
Yes, but in `plc4x-server` this is the only driver that is needed at compile time too, not just runtime, because this application uses types from that driver.

Reply:
Odd ... why is this driver needed at compile-time?

Reply:
LOL ... forgett it ... stupid me ;) ... It#s the PLC4X server, which is the server host for the PLC4X client driver. Of cours does it require these types. I was sort of thinking of the OPC-UA Server module instead ... please ignore my mumbling ;-)